about/team/index.md

```diff
@@ -2,61 +2,109 @@
-  written the initial GTK+ port and has overseen its development since then.
-
-* [Dr. Vadim Zeitlin](mailto:vadim@wxwidgets.org) - Vadim is a lynch-pin of the
-  wxWidgets project, working on most of the ports and responsible for some
-  monumental reengineering work and many important new classes. He also wrote
-  most of the wxUniversal widget set and maintains the
-  [Mahogany](http://mahogany.sf.net/) mail client. Vadim contributes
-  prolifically to the mailing lists, both helping 'newbies' and discussing
-  complex C++ and design issues. Vadim is based in France and runs his own
-  consultancy company, [TT-Solutions](http://www.tt-solutions.com/).
-
-* [Dr. Stefan Csomor](mailto:csomor@advancedconcepts.ch) - Another medical
-  doctor, Stefan Csomor wrote and maintains the wxMac, the port to OS X. He
-  runs [Advanced Concepts AG](http://www.advanced.ch/), a Swiss software
+* [Dr. Vadim Zeitlin](mailto:vadim@wxwidgets.org) - Vadim is the author of big
+  parts of, and maintainer of, wxMSW port, global coordinator of the wxWidgets
+  project and responsible for making new releases (or guilty of not making
+  them in time, depending on the point of view). Vadim is based in France and
+  runs his own consultancy company, [TT-Solutions](https://www.tt-solutions.com/).
+
+* [Dr. Stefan Csomor](mailto:csomor@advancedconcepts.ch) - An actual medical
+  doctor, Stefan Csomor wrote and maintains wxMac, the port to OS X, and wxiOS.
+  He runs [Advanced Concepts AG](http://www.advanced.ch/), a Swiss software
   consultancy specialising in medical software.
 
+* [Paul Cornett](mailto:wxml-paul@bullseye.com) - Paul is the maintainer of
+  wxGTK.
+
+* [Robin Dunn](mailto:robin@alldunn.com) - Robin works on the very popular
+  [wxPython](http://wxpython.org/) project, which allows using wxWidgets from
+  Python, and also contributed many fixes and improvements to wxWidgets
+  itself.
+
+* [Václav Slavík](mailto:vaclav.slavik@matfyz.cz) - Václav worked on wxHTML, the
+  XRC system, the bakefile-based build system and many other areas. He is also
+  the author of [Poedit](https://poedit.net/), the powerful and intuitive
+  translation editor.
+
+* [Bryan Petty](mailto:bryan@ibaku.net) - Bryan contributed to several areas
+  of wxWidgets and has taken care of hosting wxTrac (now retired), wxWiki and
+  wxWidgets documentation site, without speaking of this website itself, for
+  many years.
+
+
+## Other Active Developers
+
+Thanks to the following people for their continued contributions to wxWidgets:
+
+* Artur Wieczorek who is the maintainer of wxPropGrid library and the resident
+  graphics expert.
+
+* Maarten Bent who is a very regular contributor to wxMSW, implementing big
+  parts of high DPI support in it, maintainer of the CMake-based build system
+  and much more.
+
+* Tobias Taschner who has added several entire components of wxWidgets,
+  including, but not limited to, `wxWebRequest` and the related classes, and
+  `wxWebViewEdge` backend.
+
```

Comment:
```suggestion
* Tobias Taschner who has implemented the CMake-based build system and
  added several entire components of wxWidgets, including, but not limited
  to, `wxWebRequest` and the related classes, and `wxWebViewEdge` backend.

```

Reply:
Oops, sorry for forgetting to mention this and thanks for the reminder!